index.php

```diff
@@ -1,22 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+</head>
+<body>
+    
 <?php
+require __DIR__ . '/vendor/autoload.php';
 
-$curl = curl_init();
+$client = new GuzzleHttp\Client;
 
-// curl_setopt_array($curl, array(
-//     CURLOPT_URL => "https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services" ,
-//     CURLOPT_HTTPHEADER => array(
-//         "Content-Type: text/plain"
-//     ),
-// ));
+for($i=15;$i>=12;$i--){
 
-$response = curl_exec($curl);
-$response = json_decode($response,true);
- echo $response->data->attributes->body;
-//  var_dump($response);
-curl_close($curl);
+$response = $client->request("GET","https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services");
 
+$res = $response->getBody();
 
+$res = json_decode($res);
 
-echo $curl;
+// var_dump($res);
 
-?>
\ No newline at end of file
+$data = $res->data[$i]->relationships->field_image->links->related->href;
+
+// $client = new Client(['base_uri' => 'https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services']);
+
+// var_dump($data);
+
+// echo $data;
+
+$image = $client->request("GET","$data");
```

Comment:
Why we are adding extra quotes in $data?

Reply:
removed quotes